src/yaml/to_yaml.cr

```diff
@@ -193,24 +193,13 @@ module Enum::ValueConverter(T)
   # used for serialization.
   #
   # ```
-  # enum Stages
```

Comment:
Sorry, i just noticed that here there are also some definitions that should stay in order to make them self-contained.

Reply:
Oh, I missed to revert the yaml one. 😃